### DISCUSSION TITLE
How to isolate one monorepo package's one module completely

### DISCUSSION THREAD

**Discussion by rikurainio** [pnpm/pnpm#7859] [General]
Hey.

I am having two packges in my project:

packages:
/example-app
/helper-app

These both have a module called better-sqlite3 in their own node_modules.
I want them to have their own isolated versions of this module, since they need to be built
differently for each package.

How can I achieve this?

**Comment by michaelschufi** [comment]
I'm having a similar issue and would love to hear what's the best practice. I tried it with disabling any hoisting besides the react types package:
```
// .npmrc in the monorepo root
hoist-pattern[]='@types/react'
```
I'm on version 8.15.5.
